tests/entities_column_value_readonly_tests.py

```diff
@@ -0,0 +1,93 @@
+import json
+from nose.tools import ok_, eq_, raises
+
+from moncli import entities as en, error as e
+from moncli.enums import *
+
+def test_should_create_column_value_with_empty_list_value_using_no_input_api_data():
```

Comment:
There are some merge conflicts that need to be resolved.

Also, please name these according to the type that is being tested.

Ex. test_should_create_file_column... etc.